cmd/manager/resultcollector.go

```diff
@@ -385,7 +414,7 @@ func getMutualHttpsTransport(c *scapresultsConfig) (*http.Transport, error) {
 // an exit code of 2 means that the scan returned non-compliant
 // an exit code of 1 means that the scan encountered an error
 func exitCodeIsError(exitcode string) bool {
-	return exitcode != "0" && exitcode != "2"
+	return exitcode != common.OpenSCAPExitCodeCompliant && exitcode != common.OpenSCAPExitCodeNonCompliant
```

Comment:
Why not just `return exitcode == common.OpenSCAPExitCodeError` ?

Reply:
Because I was planning to re-use the error output for something else. The initial patch was actually using an exit error code 3. I still think that being able to re-use that will be useful in the future.